Add uniswap v3 queries for swaps (#2894)
rotki/rotki: docs/changelog.rst

=========

* :feature:`1549` Rotki premium users will now be able to switch to a dark mode and change the theme colors.

> * :feature:`2844` Users will now have available their swaps from Uniswap V3.

Reviewer: ```suggestion
* :feature:`2844` Premium users will now be able to fetch their Uniswap v3 swaps
```
